README.md

```diff
@@ -1,65 +1,31 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
 
-> One paragraph statement about the project.
+# Hello Microverse
 
-![screenshot](./app_screenshot.png)
+> Module 1, Day 3. MyPortfolio Activity.
 
-Additional description about the project and its features.
+![screenshot](./app_screenshot.png) <!--- Place a screenshot of my project here -->
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+<!--- This is an HTML comment in Markdown -->
 
-## Live Demo
+Learning how to use README file to show project information. 
 
-[Live Demo Link](https://livedemo.com)
+## Built With
 
+- HTML, CSS
 
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
```

Comment:
- [ ] Please consider adding instructions on how to get your project running locally. You can check this [article](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) to learn how to do this.